modules/administration-usage-page-overview.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="administration-usage-page-overview_{context}"]
+= Administration usage page overview
+
+When you click *Show administration usage* in the *System Health* page, you can view the product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from Sensors. This information is useful to estimate {rh-rhacs-first} consumption data for reporting.
```

Comment:
Rather than making a claim about usefulness (subjective), should probably state it as what the user can do
```suggestion
When you click *Show administration usage* in the *System Health* page, you can view the product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from Sensors. You can use this information to estimate {rh-rhacs-first} consumption data for reporting.
```